### PR TITLE
Work and about me section of mobile project

### PR DESCRIPTION
**Pull request by peteryeungtt**
Create work and "about me" session using HTML and CSS mobile version using this branch

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -99,18 +100,303 @@ html {
...
+
+.see-work-btn:focus {
+  background-color: var(--color-secondary);
+  color: var(--color-white);
+}
+
+.see-work-btn:disabled {
+  background-color: var(--color-white);
+  color: #5b5957;
+}
```
Comment (Kamba56) [inline styles.css]: - [x] Great job adding the hover effect. Kindly make your hover and active state match the project template. You are expected to stick to the template as much as possible.
image

File: index.html
Comment (Kamba56) [inline index.html:1]: - [x] Kindly use an indentation of two spaces in all your documents. This is a convention and also best pracrice.

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @peteryeungtt, 

Good job so far!!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] Great job using grid and flexbox to layout contents👏👏👏
- [x] You really did a good job building this project, keep up the good work

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (peteryeungtt) [reply styles.css:204]: @Kamba56 I didnt understand your comment fully, can you please explain more in details

File: index.html
Comment (peteryeungtt) [reply index.html:1]: @Kamba56 I didnt understand your comment fully, can you please explain more in details

Comment (peteryeungtt) [reply index.html:1]: I am using prettier vscode extension, it suppose do indentation automatically

File: README.md [hunk 2]
```diff
@@ -74,21 +76,25 @@ After you're finished please remove all the comments and instructions!
...
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
-<!-- GETTING STARTED -->
+## 🚀 Live Demo <a name="live-demo"></a>
 
-## 💻 Getting Started <a name="getting-started"></a>
+- [Live Demo Link](https://rawcdn.githack.com/peteryeungtt/My-Portfolio/b34888391911b4f667e807021bd5a65a2edfa0e4/index.html)
```
Comment (ademibowale) [inline README.md]: - [ ] [Optional] As the project not yet completed and its not in the project requirement to deploy incomplet project please i would suggest you to remove the demo link .

**Review by ademibowale — APPROVED** [review]

**Review by ademibowale — APPROVED** [review]
Hi @peteryeungtt  , 


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good use of Github flow ✔️






- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (peteryeungtt) [reply README.md]: @ademibowale I was not suppose to be a full link, but a demo link

**Pull request merged**